<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createDerived1Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createDerived2Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Створити</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Derived 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Derived 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListDerived1Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Відобразити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showListDerived2Button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Відобразити</string>
    </property>
   </widget>
   <widget class="QListView" name="listWidgetDerived1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listWidgetDerived2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
